cmake/modules/FindLibCXX.cmake

```diff
@@ -0,0 +1,15 @@
+# Find libc++
+
+find_library(LibCXX_LIBRARY NAMES c++)
+find_library(LibCXXABI_LIBRARY NAMES c++abi)
```

Comment:
Why do we care about finding c++abi?  It may be statically linked into c++, or it may be an alternative library like libsupc++ or libcxxabi or even libcxxrt.